src/registerToolbar.ts

```diff
@@ -89,6 +89,7 @@ export default function registerToolbar() {
         icon: "fa-trash-alt",
         group: ToolbarItemGroup.Regions,
         type: ToolbarItemType.Action,
+        accelerators: ["Ctrl+Delete"],
```

Comment:
should put this in a different PR, I prefer not mixing code

Reply:
Cool. Will do